### PR TITLE
WQP pipeline for secchi

### PR DESCRIPTION
**Pull request by jordansread**
@lawinslow or @mhines-usgs can you take a look?

need to finish up temperature, but the pattern is there (need a munge_temperature function)

### CODE REVIEW

File: remake_wqp.yml [hunk 1]
```diff
@@ -48,3 +50,10 @@ targets:
     
   secchi.clean:
     command: munge_wqp(secchi.merged)
+    depends: 
+      - scripts/download_munge_wqp.R
```
Comment (lawinslow) [inline remake_wqp.yml]: I don't think you need the explicit depends here as your sources are up above, do you? I think if the function changes, it's re-run.

File: remake_wqp.yml [hunk 2]
```diff
@@ -48,3 +50,10 @@ targets:
...
+    
+  ../lakeattributes/data-raw/secchi/wqp_secchi.tsv:
+    command: map_wqp(secchi.clean, wqp.nhd.lookup, target_name)
+    depends:
+      - scripts/download_munge_wqp.R
```
Comment (lawinslow) [inline remake_wqp.yml]: same here

File: remake_wqp.yml (hunk 1 above)
Comment (jordansread) [inline remake_wqp.yml]: wasn't sure - I added that and it re-ran after a change vs not before...will figure it out.

Comment (lawinslow) [inline remake_wqp.yml]: Just tried it on mine, function changes are being caught on my end.

**Comment by jordansread** [comment]
updated. thx @lawinslow .

**Pull request merged**